conda/cli/conda_argparse.py

```diff
@@ -361,6 +368,27 @@ def __call__(self, parser, namespace, values, option_string=None):
 # #############################################################################################
 
 
+def configure_parser_plugins(sub_parsers, plugin_subcommands):
+    for plugin_subcommand in plugin_subcommands.values():
+        parser = sub_parsers.add_parser(
+            plugin_subcommand.name,
+            description=plugin_subcommand.summary,
+            help=plugin_subcommand.summary,
+            formatter_class=RawDescriptionHelpFormatter,
+        )
+        # we store all other arguments here, so we can pass them to the
+        # plugin subcommands later
+        parser.add_argument(
+            "plugin_args",
+            nargs=argparse.REMAINDER,
+            help=argparse.SUPPRESS,  # to hide it from the help output
+        )
+        try:
+            plugin_subcommand.configure_parser(parser)
+        except NotImplementedError:
+            pass
```

Comment:
I think this collection of remaining arguments should only occur if the plugin doesn't define `configure_parser`:

```suggestion
        try:
            plugin_subcommand.configure_parser(parser)
        except NotImplementedError:
            # we store all other arguments here, so we can pass them to the
            # plugin subcommands later
            parser.add_argument(
                "plugin_args",
                nargs=argparse.REMAINDER,
                help=argparse.SUPPRESS,  # to hide it from the help output
            )
```

Reply:
I also think we need to disable `argparse`'s default `--help` argument when we enable this `plugin_args` argument since we'd want the plugin to intercept the `--help` argument and handle it as they see fit

Reply:
I've switched the argument parser to handle option conflicts more gracefully, which should suffice to make this possible.